### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by jonkwheeler** [LWC-Essentials/storybook#5]
@priandsf is this project dead?

I'm looking for the latest way to use LWC with Storybook. Can you help? Thanks.

**Comment by mmorozov-noltic** [comment]
Looks like, but I built it successful and plan to use in my project)

**Comment by jonkwheeler** [comment]
you got a repo I can peek at?

**Comment by mmorozov-noltic** [comment]
@jonkwheeler If you ask me - no((. This project does not support standard SF components(((. So I will use LWR

**Comment by jonkwheeler** [comment]
@mmorozov-noltic are you using LWR + Storybook? I was going to attempt to integrate it. Clearly you can since https://lwc.studio/ is using both LWC and Storybook.

**Comment by mmorozov-noltic** [comment]
Yep, I am planing use LWR + Storybook. https://lwc.studio/ - looks interest. I will take a look! Thanks!

**Comment by jonkwheeler** [comment]
@mmorozov-noltic to confirm... you are **NOT** currently using LWR + Storybook, correct?

**Comment by mmorozov-noltic** [comment]
right now - not, just planning. No have time for it now(((
